pom.xml

```diff
@@ -166,6 +166,11 @@
       <artifactId>folio-di-support</artifactId>
       <version>1.0.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>folio-holdingsiq-client</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
```

Comment:
please update to released version when available

Reply:
will do right after the release